### ISSUE TITLE
Bindable bug

### ISSUE THREAD

**Issue by JeroenVinke** [aurelia/dialog#64]
I believe there is a bug in the dialog plugin where `@bindable answer` does not trigger `answerChanged()` when directly in the dialog. @bindables work fine for custom elements inside the modal though

I have reproduced this with the latest skeleton [here](https://github.com/JeroenVinke/dialog-bug). If you remove [this line](https://github.com/JeroenVinke/dialog-bug/blob/master/src/prompt.js#L16) you even get an error when you open the dialog:

``` 
Unhandled promise rejection TypeError: Cannot read property 'getValue' of undefined
    at Prompt.descriptor.get [as answer] (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:2933:49)
    at AccessScope.evaluate (http://192.168.178.23:9001/jspm_packages/github/aurelia/binding@0.11.3/aurelia-binding.js:1145:21)
    at Binding.bind (http://192.168.178.23:9001/jspm_packages/github/aurelia/binding@0.11.3/aurelia-binding.js:4305:36)
    at View.bind (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:833:21)
    at View.bind (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:848:21)
    at View.bind (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:848:21)
    at Controller.bind (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:2747:19)
    at Controller.automate (http://192.168.178.23:9001/jspm_packages/github/aurelia/templating@0.17.4/aurelia-templating.js:2701:12)
    at http://192.168.178.23:9001/jspm_packages/github/aurelia/dialog@0.4.4/dialog-service.js:63:28
    at h (http://192.168.178.23:9001/jspm_packages/npm/core-js@1.2.6/client/shim.min.js:1773:66)
```

![image](https://cloud.githubusercontent.com/assets/2189477/11169811/58fc8ac2-8bc2-11e5-9dd6-c6801faf8ebd.png)

I would expect to also see `prompt: answer changed`

**Comment by JeroenVinke** [comment]
I can't reproduce it in the [plunkr](http://plnkr.co/edit/MMBXIF?p=info) though, but since it uses `0.4.1` and not the latest (`0.4.4`) I believe this may only occur in the latest versions of aurelia-dialog

**Comment by JeroenVinke** [comment]
OK, I think I traced it down. I'm pretty sure `ensureViewModel` of the `templating` library is not copying the properties array from `context` into the new behavior it creates. this happens [here](https://github.com/aurelia/templating/blob/ac02cb7d5fa1de57bdfe5b0eb480543673a22afb/src/composition-engine.js#L143-L148)

![img4](https://cloud.githubusercontent.com/assets/2189477/11194780/5ea6e7f0-8cae-11e5-8720-74815d8fed76.PNG)

![img5](https://cloud.githubusercontent.com/assets/2189477/11194781/60b5059a-8cae-11e5-86f0-46e4289faf34.PNG)

even though `context` contains the `BindableProperty` in the `properties` array, it doesn't exist in the new HtmlBehaviorResource [here](https://github.com/aurelia/templating/blob/ac02cb7d5fa1de57bdfe5b0eb480543673a22afb/src/composition-engine.js#L146)

**Comment by EisenbergEffect** [comment]
I think that's it! Nice work tracking this down. I'll take this one.

**Comment by JeroenVinke** [comment]
I have just confirmed that the changes in the commit fixes the issue I had, thanks

**Comment by EisenbergEffect** [comment]
Excellent! Thanks for the confirmation. This will get released today or tomorrow.

**Comment by LivesleyWA** [comment]
Can also confirm this issue is now fixed with 0.5.1 release. Happy days - my dialogs work again!
